src/Game/AccountManagement.java

```diff
@@ -356,8 +1034,153 @@ public static void account_panels_input_hook(int n1, int mouseY, int a, int n3,
+					cursorIndex = 0;
+					
+					while (true) {
+						if (cursorIndex >= 5) {
+							for (int outerIdx = 0; outerIdx < 5; ++outerIdx) {
+								String checkedAnswer = Panel.getControlText(Client.panelRecoveryQuestions, Client.controlAnswerInput[outerIdx]);
+
+								for (int innerIdx = 0; innerIdx < outerIdx; ++innerIdx) {
+									String questionAnswer = Panel.getControlText(Client.panelRecoveryQuestions, Client.controlAnswerInput[innerIdx]);
+									if (checkedAnswer.equalsIgnoreCase(questionAnswer)) {
+										Panel.setControlText(Client.panelRecoveryQuestions,Client.controlRecoveryQuestions, "@yel@Each question must have a different answer");
+										return;
+									}
+								}
+							}
+
+							sendRecoveryQuestions();
+
+							for (int questionIndex = 0; questionIndex < 5; ++questionIndex) {
+								recoveryIndices[questionIndex] = questionIndex;
+					            Client.controlRecoveryText[questionIndex] = recoveryQuestions[recoveryIndices[questionIndex]];
+					            Panel.setControlText(Client.panelRecoveryQuestions,Client.controlAnswerInput[questionIndex], "");
+					            Panel.setControlText(Client.panelRecoveryQuestions,Client.controlRecoveryIns[questionIndex], questionIndex + 1 + ": " + Client.controlRecoveryText[questionIndex]);
+							}
+
+							Client.clearScreen();
+				            Client.showRecoveryQuestions = false;
+							break;
+						}
+
+						String chkAnswer = Panel.getControlText(Client.panelRecoveryQuestions, Client.controlAnswerInput[cursorIndex]);
+						if (chkAnswer == null || chkAnswer.length() < 3) {
+			                Panel.setControlText(Client.panelRecoveryQuestions,Client.controlRecoveryQuestions, "@yel@Please provide a longer answer to question: " + (cursorIndex + 1));
+			                return;
+			            }
+
+						++cursorIndex;
+					}
+				}
+		  }
+	  } catch (Exception e) {
+	      e.printStackTrace();
+	    }
+  }
+  
+  public static void contact_details_input(int n1, int mouseY, int n3, int mouseX) {
+	  try {
+		  Panel.handleMouse(Client.panelContactDetails, n1, mouseY, n3, mouseX);
+	        if (Panel.isSelected(Client.panelContactDetails, Client.fullNameInput)) {
+	        	Panel.setFocus(Client.panelContactDetails, Client.zipCodeInput);
+	        }
+	        if (Panel.isSelected(Client.panelContactDetails, Client.zipCodeInput)) {
+	        	Panel.setFocus(Client.panelContactDetails, Client.countryInput);
+	        }
+	        if (Panel.isSelected(Client.panelContactDetails, Client.countryInput)) {
+	        	Panel.setFocus(Client.panelContactDetails, Client.emailInput);
+	        }
+	        if (Panel.isSelected(Client.panelContactDetails, Client.emailInput)) {
+	        	Panel.setFocus(Client.panelContactDetails, Client.fullNameInput);
+	        }
+	        if (!Panel.isSelected(Client.panelContactDetails, Client.chooseSubmitContactDetailsButton)) return;

```

Comment:
I would prefer this not to be expressed as a one-liner; would be easier to read if it were just normal code block
```java
 if (Panel.isSelected(Client.panelContactDetails, Client.chooseSubmitContactDetailsButton)) {
...
}
```

Reply:
fixed